Monet_Harkin/FullFriends/mysqlconnection.py

```diff
@@ -0,0 +1,39 @@
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.sql import text
+# Create a class that will give us an object that we can use to connect to a database
+class MySQLConnection(object):
+    def __init__(self, app, db):
+        config = {
+                'host': 'localhost',
+                'database': db, # we got db as an argument
+                'user': 'root',
+                'password': 'MonetVG12',
```

Comment:
I hope you're not using this password anywhere else since the whole world is able to see it